airflow/plugins_manager.py

```diff
@@ -40,11 +43,29 @@
 import_errors = {}
 
 
+PY37 = sys.version_info >= (3, 7)
+
+
 class AirflowPluginException(Exception):
     pass
 
 
-class AirflowPlugin(object):
+class _MetaPluginClass(type):
+    def __new__(cls, name, bases, props):
+        if props.get('operators', []) or props.get('sensors', []):
+            warnings.warn(
+                "Registering operator or sensors in plugins is deprecated -- these should be treated like "
+                "'plain' python modules, and imported normally in DAGs. Airflow 2.0 has removed the ability "
+                "to register these types in plugins",
```

Comment:
```suggestion
                "Registering operators or sensors in plugins is deprecated -- these should be treated like "
                "'plain' python modules, and imported normally in DAGs. Airflow 2.0 has removed the ability "
                "to register these types in plugins",
```

Looks good for now, maybe we can link it to Module Management page once we backport that doc PR to v1-10-test https://airflow.readthedocs.io/en/latest/modules_management.html

Reply:
http://airflow.apache.org/docs/stable/howto/custom-operator.html exists on 1.10 already, and is a page all about this, so I'll use that instead.

Reply:
👍

Reply:
Done, check out new message